cts/cli/regression.crm_mon.exp

```diff
@@ -3466,38 +3466,33 @@ Operations:
 =#=#=#= Begin test: Text output of partially active resources =#=#=#=
 Cluster Summary:
   * Stack: corosync
-  * Current DC: cluster02 (2) (version) - partition with quorum
+  * Current DC: cluster02 (version) - partition with quorum
```

Comment:
The regression tests shouldn't change, and I think that's why CI is failing -- you should just be able to drop these changes